src/main/java/com/lmax/disruptor/RingBuffer.java

```diff
@@ -24,14 +27,39 @@
  *
  * @param <E> implementation storing the data for sharing during exchange or parallel coordination of an event.
  */
-public final class RingBuffer<E> implements Cursored, DataProvider<E>
+abstract class RingBufferPad
 {
-    public static final long INITIAL_CURSOR_VALUE = Sequence.INITIAL_VALUE;
+    protected long p1, p2, p3, p4, p5, p6, p7;
```

Comment:
Hi  mikeb01,We know that
class Sequence{
	protected long p1, p2, p3, p4, p5, p6, p7;
	protected volatile long value;
	protected long p9, p10, p11, p12, p13, p14, p15;
}
the p1~p15 in Sequence class above is used to protect "value" field from false sharing,I can understand this well.
But I can understand which field(or fields) the padding in the RingBuffer.java protect from false sharing. Thanks!